shade-test/jar-file/src/test/java/org/testcontainers/JarFileShadingTest.java

```diff
@@ -0,0 +1,79 @@
+package org.testcontainers;
+
+import org.assertj.core.api.ListAssert;
+import org.junit.*;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.*;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JarFileShadingTest {
+
+    private static FileSystem fileSystem;
+
+    private static Path root;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        Path path = Paths.get("..", "..", "core", "target", "testcontainers-0-SNAPSHOT.jar");
```

Comment:
Then how does it matter where the jar file is? Again, the test will fail if we change paths. 

Doing "copy dependency" might lead to the dependency copied from Maven's local repo => **more magical and error-prone**